CHANGELOG.md

```diff
@@ -71,7 +71,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     * `yuyo.components.with_static_button` to `.with_static_interactive_button`
     * `yuyo.components.with_static_channel_select` to `.with_static_channel_menu`
 
-
```

Comment:
Add a note about changing the prefix matching system to always use ":" as a splitter for the component client